src/vscode-bicep/src/deploy/azure-account.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { Event } from "vscode";
+
+// https://github.com/microsoft/vscode-azure-account/blob/main/src/azure-account.api.d.ts
+// with just the properties we need
+export interface AzureAccount {
```

Comment:
The convention is to use the file name `types.ts` for ts files that only export types. I would also recommend that we move this file and `azureClients.ts` to `src/azure`. We may add other features (e.g., #3915) in the future that will be dependent on the azure client and types.

Reply:
Sure, updated here:
https://github.com/Azure/bicep/commit/5a3f292f2c4e3108ab60f9d1aa9466206980b788

Thanks!